src/main/java/io/turntabl/Polls/models/UserTO.java

```diff
@@ -0,0 +1,55 @@
+package io.turntabl.Polls.models;
+
+public class UserTO {
```

Comment:
is this a user or a creator?
if it's a creator let's call it that
if it can be someone who submits a vote, let's rename the members

Reply:
creator isn't needed at the moment, it has been removed